iq_group.info.yml

```diff
@@ -2,7 +2,7 @@ name: iQual Group Integration module
 description: Integration module for Group
 package: Custom
 type: module
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: 8.8 - 9.4
```

Comment:
Is there a range option? I can only find ">=8.8" or similar.

Reply:
Semantic versioning is supported according to this https://www.drupal.org/node/3070687
And that allows ranges, yes

Reply:
Thanks for the link 👍 All good then :)